entries/iteration.qmd

```diff
@@ -22,22 +22,24 @@ Based on these inspirations, an iteration for us follows this general sequence o
     - Will add the aim, output, as well as the end date of the iteration to be discussed and decided on in the first meeting.
     - Will start adding some tasks into the board.
-2. The first planning meeting for the iteration is scheduled well in advance, for between 1-2 hours and right after the previous iteration's debrief. Preferably this meeting is in person.
+2. The first planning meeting for the iteration is scheduled well in advance, for between 1-2 hours and (preferably)right after the previous iteration's debrief. Preferably this meeting is in person.
     - The aim of the meeting to discuss and plan the specifics of the iteration.
     - Before the planning meeting, team members will review the aim and output of the iteration board and write out potential tasks (as GitHub Issues or as drafts on the board) needed to complete the aim.
-    - During the meeting, we'll discuss and decide on the end output (the "increment" or "milestone").
-    - During the meeting, we'll brainstorm and write out more tasks that are needed to complete the increment or milestone, or select them from the existing list of tasks. 
-    - Each team member self-assigns to a task (self-assign to Issue) and estimates the time spent on the tasks by using the Iteration's Roadmap Board. 
-    - We'll review whether the number of tasks, time estimation of those tasks, and the aim of the iteration are realistic in the time available.
-    - We'll discuss and agree on how long the iteration should be or will take and set a date for the iteration's last meeting.
+    - During the meeting, we'll discuss and decide or agree on the end output (the "increment" or "milestone").
+    - During the meeting, we'll brainstorm and write out more tasks that are needed to complete the increment or milestone, or select them from the existing list of tasks. Preferably someone will share their screen (if virtual) and the team goes through the tasks together.
+    - While making tasks (Issues), keep them as small as is reasonable and as descriptive and targeted as possible.
+    - As we discuss each task, we collectively assign team members to tasks (assign to the Issues).
+    - We'll review the number of tasks and the aim of the iteration, then discuss and agree on how long the iteration should be (can range from 2-6 weeks, but prefer shorter iterations over longer ones).
+    - After the iteration length is decided, we'll schedule an approximately 2 hour debrief/retrospective meeting on the end date.
+    - Either during or afterward the planning meeting, each team member individually estimates how much time they might need on their assigned tasks, taking into consideration how much time they have available during their work on the iteration. They'll use the the Iteration's Roadmap Board to track and estimate their time spent on tasks.
```

Comment:
Oh, so we will use the roadmap board to track and estimate time? Not add the field "Estimated hours"?